external/fv3fit/fv3fit/_shared/models.py

```diff
@@ -62,24 +65,33 @@ def predict(self, X: xr.Dataset) -> xr.Dataset:
         return xr.merge([base_prediction, derived_prediction])
 
     def dump(self, path: str):
-        raise NotImplementedError(
-            "no dump method yet for this class, you can define one manually "
-            "using instructions at "
-            "http://vulcanclimatemodeling.com/docs/fv3fit/derived_model.html"
-        )
+        base_model_path = os.path.join(path, self._BASE_MODEL_SUBDIR)
+        options = {
+            "additional_input_variables": self._additional_input_variables,
+            "derived_output_variables": self._derived_output_variables,
+            "model": base_model_path,
+        }
+        self._base_model.dump(base_model_path)
```

Comment:
I think you need to use `fv3fit.dump` here so that it includes the name label. If you add a dump/load test, it should catch this.